website/pages-parts/LandingPage/Testimonials.tsx

```diff
@@ -70,17 +93,23 @@ export function Testimonials() {
       <Heading>Testimonials</Heading>
       <div className={styles.testimonials}>
         <Testimonial
-          company="Red Hat"
-          description="uniforms it’s one and the only library that allows you to have greater flexibility on top of the React platform to building forms you like, with great robustness with advanced support of many different forms inputs."
-          position="Team Lead"
-          who="Wojtek Trocki"
+          company="Resolve"
+          description="Uniforms is the backbone of our data-intensive web-applications. We have about 200 different forms, from very simple ones, to ones that are filled with complex data-loading conditional form components, which create an incredible UX for our users. And if you really need to push the limits of what you can do with forms, I would highly recommend <b><a href='https://www.vazco.eu/' alt='Vazco website' target='blank'>reaching out to Vazco</a></b> themselves for expert advice."
+          position="CTO and Co-Founder at Resolve"
+          who="Florian Bienefeldt"
+          avatar="img/florian-carre.jpg"
+          linkGithub="https://github.com/Floriferous"
+          linkLinkedin="https://ch.linkedin.com/in/florianbienefelt"
```

Comment:
1. The image name doesn't match the name.
2. The name doesn't match the LinkedIn link.
3. Let's move all avatars into `img/avatar/{GitHub handle}` instead.
4. It'd be better to have all avatars in the same format (`.jpg` is fine).

Reply:
Details matter. It is now more systematic.